lib/wallet/test/unit/Cardano/Wallet/Address/Discovery/RandomSpec.hs

```diff
@@ -8,6 +8,7 @@
 {-# LANGUAGE TypeApplications #-}
 
 {-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
```

Comment:
I think that this warning is somewhat out of scope for this pull request. 😅  But it's done now.

Reply:
it's for the specs to use partial functions ?

Reply:
Yes. (I'm not fond of that warning.) What I meant was that the specs worked before without this `GHC_OPTIONS` pragma, but now they seem to need it, and this particular change seems out of scope to me as it touches a lot of modules.

But they have been touched now, I'm fine with proceeding.